examples/live_tiffs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Loads and Displays tiffs as they get generated in the specific directory.
+Trying to simulate the live display of data as it gets acquired by microscope. 
+"""
+
+import os
+import sys
+import time
+import numpy as np
+from skimage.io.collection import alphanumeric_key
+from dask import delayed
+import dask.array as da
+from tifffile import imread
+import napari
+from napari.qt import thread_worker
+
+
+with napari.gui_qt():
+    viewer = napari.Viewer(ndisplay=3)
+    # pass a directory to monitor or it will monitor current directory.
+    path = sys.argv[1] if len(sys.argv) > 1 else '.' 
+    path = os.path.abspath(path)
+    end_of_experiment = 'final.log'
+
+    def append(delayed_image):
+        if delayed_image is None:
+            return
+
+        if viewer.layers:
+            layer = viewer.layers[0]
+            image_shape = layer.data.shape[1:]
+            image_dtype = layer.data.dtype
+            dask_image = da.from_delayed(
+                delayed_image,
+                shape=image_shape,
+                dtype=image_dtype,
+            ).reshape((1,) + image_shape)
+            layer.data = da.concatenate(
+                (layer.data, dask_image), axis=0
+            )
+        else:
+            image = delayed_image.compute()
+            dask_image = da.from_delayed(
+                delayed_image,
+                shape=image.shape,
+                dtype=image.dtype,
+            ).reshape((1,) + image.shape)
+            layer = viewer.add_image(dask_image, rendering='attenuated_mip')
+
+        if viewer.dims.point[0] >= layer.data.shape[0] - 2:
+            viewer.dims.set_point(0, layer.data.shape[0] - 1)
```

Comment:
I like moving the slider to always display the most recent image, but why do you need an if statement here?

Reply:
@GenevieveBuckley it's super annoying to have the slider move to the end if you are scrolling through the middle of the stack. This is akin to terminal output scrolling only when you are at the bottom, but not if you scroll back to check something in the middle of the output.

@zeroth probably worth adding a comment above the if statement explaining this.